store/utils.js

```diff
@@ -1,3 +1,79 @@
+import camelcaseKeysDeep from 'camelcase-keys-deep'
+import Swagger from '@aeternity/aepp-sdk/es/utils/swagger'
+import swag from '../swagger.json'
+
+export const initMiddleware = () => {
+  swag.paths = {
+    ...swag.paths,
+    'name/auction/{name}': {
+      get: {
+        operationId: 'getAuctionInfoByName',
+        parameters: [
+          {
+            in: 'path',
+            name: 'name',
+            required: true,
+            type: 'string'
+          }
+        ]
+      }
+    },
+    'txs/backward': {
```

Comment:
Please, ensure that the middleware team is aware of the lack of these endpoints definitions.

Reply:
There is no lack of this endpoints, the problem in that, an array in axios request on the Swagger side converting the array type of parameter inside of url request to something that mdw does not  expect `type_group[]=...`